appium/webdriver/webdriver.py

```diff
@@ -295,8 +295,11 @@ def _update_command_executor(self, keep_alive: bool) -> None:
         executor = f'{protocol}://{hostname}:{port}{path}'
 
         logger.debug('Updated request endpoint to %s', executor)
-        # Override command executor
-        self.command_executor = RemoteConnection(executor, keep_alive=keep_alive)
+        # Override command executor.
+        if isinstance(self.command_executor, AppiumConnection):  # type: ignore
```

Comment:
why should we ignore mypy here?

Reply:
I need to take a look later as a separate PR (so minimize each file change). For now, the below error kept rising without `ignore`. Maybe it needs a couple of lines' update with type hints.

```
appium/webdriver/webdriver.py:300: error: Cannot determine type of "command_executor"  [has-type]
```

Reply:
this looks like a mypy bug.
A similar issue has been reported here: https://github.com/python/mypy/issues/13700

Lets keep it like that for now